tests/test_annotations.py

```diff
@@ -50,6 +45,30 @@ def test_annotate_submission_with_json():
         assert new_status == expected_status
 
 
+def test_annotate_submission_with_json():
+    """Test annotation"""
+    add_annotations = {'test': 2, 'test2': 2}
```

Comment:
What about a key-value pair where value is `None`? To ensure that this item doesn't get added.